### PR TITLE
Allow descriptions to contain newlines and other whitespace

### PR DESCRIPTION
**Pull request by irenarindos**

### CODE REVIEW

**Comment by AdamBouhmad** [comment]
Looks good to me! Thanks @irenarindos

**Review by louisruch — APPROVED** [review]
Thanks for getting to this!

**Review by louisruch — COMMENTED** [review]
@kheina It seems like the tests did not run, not sure why. Also might want to add a changelog entry for this.

**Comment by kheina** [comment]
> @kheina It seems like the tests did not run, not sure why. Also might want to add a changelog entry for this.

odd, I rebased again and added a changelog entry, lets see if the new commit has the same issue

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   ([PR](https://github.com/hashicorp/boundary-ui/pull/2237))
 * ui: Add UI support for MinIO (Enterprise and HCP Boundary only)
   ([PR](https://github.com/hashicorp/boundary-ui/pull/2248))
+* Allow descriptions to contain newlines and other whitespace
```
Comment (louisruch) [inline CHANGELOG.md:49]: This should be moved up to the `Next` section

**Review by louisruch — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kheina) [reply CHANGELOG.md]: my eyes glazed over and I just added it to the first new and improved section I saw, my bad. fixed

**Review by louisruch — APPROVED** [review]
:shipit:

**Pull request merged**